plugins/modules/azure_rm_virtualnetworkgateway.py

```diff
@@ -104,12 +104,17 @@
     sku:
         description:
             - The reference of the VirtualNetworkGatewaySku resource which represents the SKU selected for Virtual network gateway.
-        default: VpnGw1
+        default: VpnGw1AZ
```

Comment:
Do not change the default value unless necessary, as this will affect the current user.
```suggestion
        default: VpnGw1
```